w3af/core/data/kb/info.py

```diff
@@ -93,7 +93,7 @@ def from_info(cls, other_info):
             raise TypeError('Info expected in from_info.')
         
         name = other_info.get_name()
-        desc = other_info.get_desc()
+        desc = other_info.get_desc(with_id=False)
```

Comment:
This seems like an interesting edge case in `from_info` that should get into master!